lib/route.js

```diff
@@ -16,7 +16,7 @@ class Route extends Loggable {
 
     // importing routes for all the modules
     importModuleRoutes() {
-        this.glob.sync('api/modules/**/route.js').forEach((file) => {
+        this.glob.sync(path.join(process.cwd(), config.dir, 'api/modules/**/route.js')).forEach((file) => {
```

Comment:
Aqui nessa linha tem como vc trocar por this.config ? Caso a gente queira fazer teste melhor, acho que fica mais facil de acessar essa variavel e trocar o valor dela.